package.json

```diff
@@ -3,6 +3,7 @@
   "version": "4.2.1",
   "description": "Extra lightweight DOM selector helper",
   "main": "index.js",
+  "types": "index.d.ts",
```

Comment:
This is not needed. TypeScript infers it automatically.

Reply:
> This is not needed. TypeScript infers it automatically.

Indeed it is.
Remove it as a commit?

Reply:
Yes